samples/app-anonymous-users/nodejs/api/server/simpleGraphClient.js

```diff
@@ -0,0 +1,73 @@
+// <copyright file="simpleGraphClient.js" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// </copyright>
+
+const { Client } = require('@microsoft/microsoft-graph-client');
+
+/**
+* This class is a wrapper for the Microsoft Graph API.
+* See: https://developer.microsoft.com/en-us/graph for more information.
+*/
+class SimpleGraphClient {
+    constructor(token) {
+        if (!token || !token.trim()) {
+            throw new Error('SimpleGraphClient: Invalid token received.');
+        }
+
+        this._token = token;
+
+        // Get an Authenticated Microsoft Graph client using the token issued to the user.
+        this.graphClient = Client.init({
+            authProvider: (done) => {
+                done(null, this._token); // First parameter takes an error if you can't get an access token.
+            }
+        });
+    }
+
+    /**
+    * Collects information about the user in the bot.
+    */
+    async getMeAsync() {
+        return await this.graphClient.api('/me').get();
+    }
+
+    async getUserPhoto() {
+        return await this.graphClient.api('/me/photo/$value').get();
+    }
+
+    /**
+    * Collects information about the SharePoint drive based on SharePoint id passed.
+    */
+    async getDriveDetails(siteId) {
+        return await this.graphClient
+            .api('/sites/'+siteId+'/drives').version('beta')
+            .get().then((res) => {
+                return res;
+            });
+    }
+
+    /**
+    * Collects information about the documents stored in SharePoint drive.
+    */
+    async getContentList(siteId, driveId) {
+        return await this.graphClient
+            .api('/sites/'+siteId+'/drives/'+driveId+'/root/children').version('beta')
```

Comment:
Why are we using beta `API`?

Reply:
The majority of the samples adhere to the beta version. 
FYI
![image](https://user-images.githubusercontent.com/117716554/221478081-8e9e1972-bf80-4613-ac4b-875562377fb0.png)